librad/src/git/include.rs

```diff
@@ -21,6 +21,8 @@ use std::{
     path::{self, PathBuf},
 };
 
+use tempfile::NamedTempFile;
```

Comment:
Ahhh that's what we were looking for @xla :smile: Remembering that we couldn't get the path of a `File`.